### DISCUSSION TITLE
Fav server, server rating

### DISCUSSION THREAD

**Discussion by reneknuvers** [jamulussoftware/jamulus#1548] [Ideas]
The number of publicly available servers is growing. Some servers perform better than others. It would be nice to have a means of making a server that performs well as a ‘favourite’. The number of favs could add up to a rating for the server.
This would help both new and long time users to build a community.

**Comment by gene96817** [comment]
This scheme would favor servers in big cities. It would be wonderful to help build a community around servers out in the suburbs or small towns too.

**Reply by reneknuvers** [reply]
Good thinking. So some thought has to be spend on to who high-favs servers are promoted - if any. Maybe based on the ‘city’ entry in the user profile?
But lets keep it simple for starters and expand later: allow a user to locally store favorite servers based on the server name and maybe IP combi.

**Comment by dcorson-ticino-com** [comment]
The problem will be that a server that is good for you may not be good for me, because of distance, Internet connectivity, etc., etc.
I do not see how this can really add useful information for other users.  The best indication I know is still ping time.

**Reply by reneknuvers** [reply]
As a personal favorites list this would make sense.

**Reply by dcorson-ticino-com** [reply]
I am actually working on adding a favorites list right now.  In a version or so it may be ready.

**Reply by gene96817** [reply]
Thinking broadly, it would be interesting to know how many minutes a week or minutes a month a server is used. Similarly how many different (unique) clients has connected to the server (by week or by monty). I think this would be a better indication of "popularity".

Of course, this is not a serious proposal. We don't have an easy way to collect this information. Yet, if I want to know which server is popular, then I want to know where all the action/jamming is happening.
